doc/nh/shared/css/html.css

```diff
@@ -92,11 +92,9 @@ HR  {
     font-weight:bold;
 }
 
-.book DIV {
+.book > DIV, .chapter, .preface {
```

Comment:
Old discrepancy between single html styles and multi-pages html styles. I have let it get aligned on multi-pages html styles.